### PR TITLE
fix: add extend flag to opionally generate config within extend key o…

### PR DESCRIPTION
**Pull request by pivotpointux**
Hi @nado1001,

Apologies, but I decided to abort the previous PR and start this new one (hopefully cleaner), where I've aimed to resolve the issues you raised - all except one, which I need some clarification on please:

"However, when I run it in this state, the format of the generated tailwind.config.js is a bit broken.
I would appreciate the following change.
Please use the sd.config.js in the [example](https://github.com/nado1001/style-dictionary-tailwindcss-transformer/blob/main/example/with-css-variables/sd.config.js) and actually run it to check."

If you could please clarify, I will aim to get this resolved.

Thanks!

### CODE REVIEW

**Comment by pivotpointux** [comment]
@nado1001 Just curious if you had an update on this one?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//README
```
Comment (nado1001) [inline README.md:1]: Please delete the comments here.

File: src/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// index.ts
```
Comment (nado1001) [inline src/index.ts:1]: Please delete the comments here as well.

File: src/types.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+//types.ts
```
Comment (nado1001) [inline src/types.ts:1]: Please delete the comments here as well.

File: src/utils.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+//utils.ts
```
Comment (nado1001) [inline src/utils.ts:1]: Please delete the comments here as well.

**Review by nado1001 — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by pivotpointux** [comment]
@nado1001 Thanks for reviewing! Changes actioned.

**Review by nado1001 — APPROVED** [review]
Thanks! We will release it as version 1.4.2 after this

**Pull request merged**